client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -89,9 +89,11 @@ public class ShuffleClientImpl extends ShuffleClient {
   private final Map<Integer, ConcurrentHashMap<Integer, PartitionLocation>> reducePartitionMap =
       JavaUtils.newConcurrentHashMap();
 
-  protected final ConcurrentHashMap<Integer, Set<String>> mapperEndMap =
+  protected final ConcurrentHashMap<Integer, Set<Integer>> mapperEndMap =
```

Comment:
could you please add a comment to mention what the key/value is?

Reply:
ok, done